### ISSUE TITLE
Gaze with respect to camera

### ISSUE THREAD

**Issue by Risskov** [TadasBaltrusaitis/OpenFace#865]
The gaze vectors and head pose rotations are in world coordinates (as it also says in the documentation), so looking at the camera gives different angles depending on the location of the face in the image.

There is a function 'GetPoseWRTCamera' that gives coordinates with respect to the camera, but I am also interested in getting the same for the gaze so looking at the camera always gives an angle close to zero. Would it be enough to change the 'GetPose' call in 'GazeEstimation' to 'GetPoseWRTCamera', or is more work needed?

**Comment by TadasBaltrusaitis** [comment]
I don't believe that would be enough. I think the easiest way to accomplish this would be to shoot a ray from the 3D eye location to the camera (at 0,0,0) and see what the angle between `gaze_absolute` and the ray is. So if a person is already looking at the camera, that angle will be 0.
